docs/api/structures/usb-device.md

```diff
@@ -0,0 +1,8 @@
+# USBDevice Object
```

Comment:
Why is this a small subset of the `WebUSB USBDevice` interface? Would it make sense to expose all the other properties too?